### PR TITLE
app: add range to basetime

### PR DESCRIPTION
**Pull request by maxd-nordic**
This commit adds a range to the basetime object.

### CODE REVIEW

**Comment by maxd-nordic** [comment]
@coderbyheart

File: tests/module/environmental/src/main.c [hunk 1]
```diff
@@ -129,22 +113,6 @@ void setUp(void)
 	zbus_sub_wait(&battery, &chan, K_NO_WAIT);
 }
 
-void test_all_zeroes(void)
```
Comment (coderbyheart) [inline tests/module/environmental/src/main.c:132]: Unrelated change. Also not mentioned in the commit.

**Review by coderbyheart — CHANGES_REQUESTED** [review]

File: tests/module/environmental/src/main.c (hunk 1 above)
Comment (maxd-nordic) [reply tests/module/environmental/src/main.c:132]: not unrelated - this test is not compatible with the change

Comment (maxd-nordic) [reply tests/module/environmental/src/main.c:132]: added this to the commit message - cannot make this a fail-expected test because the module is forcing a crash in that case.
